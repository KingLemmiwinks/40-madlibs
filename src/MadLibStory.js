import React from "react";

const MadLibStory = ({ madLibText, storyTitle, startOver }) => {
  return (
    <div className="story-container">
      <h3>{storyTitle}</h3>
      <p>{madLibText}</p>
      <button type="button" onClick={startOver}>
        Start Over
      </button>
    </div>
  );
};

export default MadLibStory;
