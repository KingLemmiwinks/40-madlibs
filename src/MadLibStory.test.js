import React from "react";
import { render } from "@testing-library/react";
import MadLibStory from "./MadLibStory";

it("will render without crashing", () => {
  render(<MadLibStory />);
});

it("matches snapshot", () => {
  const { asFragment } = render(<MadLibStory />);
  expect(asFragment()).toMatchSnapshot();
});
