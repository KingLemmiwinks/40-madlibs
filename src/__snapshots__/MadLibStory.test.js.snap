// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="story-container"
  >
    <h3 />
    <p />
    <button
      type="button"
    >
      Start Over
    </button>
  </div>
</DocumentFragment>
`;
